riotfile.py

```diff
@@ -2824,7 +2824,7 @@ def select_pys(min_version: str = MIN_PYTHON_VERSION, max_version: str = MAX_PYT
         ),
         Venv(
             name="subprocess",
-            command="pytest {cmdargs} --no-cov tests/contrib/subprocess",
+            command="pytest {cmdargs}  --no-ddtrace --no-cov tests/contrib/subprocess",
```

Comment:
Can we try to avoid this change? Can you confirm the tests would fail if we don't add this change?